FFMpegCore/FFMpeg/FFMpegArguments.cs

```diff
@@ -57,6 +57,12 @@ private FFMpegArgumentProcessor ToProcessor(IOutputArgument argument, Action<FFM
         {
             var args = new FFMpegArgumentOptions();
             addArguments?.Invoke(args);
+
+            if (argument is OutputPipeArgument && !Arguments.Any(x => x is ForceFormatArgument))
```

Comment:
I think it may make more sense to include format directly in the `.OutputToPipe(..)` call than this. 
Secondly, I'd prefer separate features as separate PRs, though I won't block this PR for that reason

Reply:
it is better to leave it as it is for better code readability

Reply:
I do not like this feature of automagically adding a forceformatargument without the user's knowledge.

Reply:
And what improves code readability?